docs/_guide/02-writing-templates.md

```diff
@@ -286,4 +286,83 @@ html`${cache(data.showDetails
 When lit-html re-renders a template, it only updates the modified portions: it doesn't create or remove any more DOM than it needs to. But when you switch from one template to another, lit-html needs to remove the old DOM and render a new DOM tree. 
 
 The `cache` directive caches the generated DOM for a given binding and input template. In the example above, it would cache the DOM for both the  `summaryView` and `detailView` templates. When you switch from one view to another, lit-html just needs to swap in the cached version of the new view, and update it with the latest data.
+### Rendering nothing
+Very often you may conditionally render something, but in the other case
+you do not want to render anything at all. So, if you are using ternary 
+operators in one case you might return your template and in the other you
+want to return no content lit-html leaves you with a couple options:
+* an empty string (`''`)
+* null
+* undefined
+* nothing sentinel
+#### An empty string
```

Comment:
Don't really need these sections. Could move the example up to the intro section if you like.

Reply:
see my comments below.